chart/openfaas/templates/ingress.yaml

```diff
@@ -21,8 +22,15 @@ spec:
         paths:
           - path: {{ $host.path }}
             backend:
+              {{- if $apiIsStable }}
```

Comment:
Oh wow, there is even more to change than I had remembered.. 

I would love a quick blog post on this recipe on openfaas.com, if you're open to writing it as a guest author?

Reply:
yeah, on top with 1.18 comes pathType :D 

I know the pain, we have OCP 3.11 at work and now 4.7 :D Migrating is my second hobby